### PR TITLE
Adds bushranger gear and room

### PR DESCRIPTION
**Pull request by MisterMecky**
## About The Pull Request

Adds two new items, a bushrangers helmet and the bushrangers armour, themed after the iconic Ned Kelly. Also adds a random maint room for the armour to spawn.

bushranger

This is a still a work in progress, things to do.

- [x] Remove austation defines from includes
- [x] Make armour appear on a corpse in the random room
- [x] Make bushranger armour not a subtype of bulletproof armor
- [x] Tweak armor values so that they are fair
- [x] Re-number random room and convert using Map Merger
- [x] Ensure everything is working properly and is looking good

## Why It's Good For The Game

Adds new australian themed content to the game.

## Changelog
:cl:
add: Bushranger armour and helmet
add: Bushranger room
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
👏

**Comment by MisterMecky** [comment]
Alright, I feel like this is good where it at now.

File: austation/code/modules/awaymissions/corpse.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+	name = "Bushranger"
+	outfit = /datum/outfit/bushranger
+
+/datum/outfit/bushranger
+	name = "Bushranger"
+	uniform = /obj/item/clothing/under/color/grey
+	suit = /obj/item/clothing/suit/armor/nedarmour
+	head = /obj/item/clothing/head/helmet/nedhelmet
+	shoes = /obj/item/clothing/shoes/sandal
+	gloves = /obj/item/clothing/gloves/color/brown
```
Comment (MisterMecky) [inline austation/code/modules/awaymissions/corpse.dm:11]: Not sure what that no newline business is all about

Comment (TheFakeElon) [reply austation/code/modules/awaymissions/corpse.dm:11]: git warns you when you don't have a newline at the end of your file

**Comment by Ohagi-Chan** [comment]
True blue

**Review by TheFakeElon — COMMENTED** [review]
the laser resistance should be stronger imo, given it's made from metal. Other than that, looks good!

**Comment by Ohagi-Chan** [comment]
hmmmmm?

**Comment by Singularity69** [comment]
?!?!?!?

**Comment by MisterMecky** [comment]
don't mind me

**Review by TheFakeElon — APPROVED** [review]

**Pull request merged**